pontoon/sync/static/js/sync_log.js

```diff
@@ -0,0 +1,33 @@
+$(function () {
+  // Remove any existing sync-status-* class
+  function removeSyncStatusClasses($element) {
+    $element.removeClass(function (index, className) {
+      return (className.match(/(^|\s)sync-status-\S+/g) || []).join(' ');
+    });
+  }
+
+  // Request to sync a project
+  $('.sync').click(function (e) {
+    e.preventDefault();
+
+    const projectSlug = $(this).attr('project-slug');
+    const $row = $(this).closest('tr');
+    const $statusCell = $row.find('td').eq(1);
+
+    removeSyncStatusClasses($statusCell);
+    $statusCell.addClass('sync-status-other').text('Requested');
```

Comment:
Why not use the "In progress" status here?

Alternatively, you could avoid all the code that manages the statuses and their classes by using something like this:

```js
Pontoon.endLoader("Sync in progress...", "error")
```

And this in case of an error:

```js
Pontoon.endLoader("Unable to sync the project.", "error")
```

I think that's a fair compromise given the change will only be visible for a second or two.

Reply:
> Why not use the "In progress" status here?

Because I cannot know if the sync actually is in progress or failed.

Going with your proposal, I assume the `error` in your first code is not needed.